vocab/properties.json

```diff
@@ -779,7 +773,7 @@
         ]
     },
     "hasParent": {
-        "description": null,
+        "description": "Reference to the source object or legal person.",
```

Comment:
"source" is rather vague here.

Reply:
I know... you have an idea how to make it more concrete but explaining "parent" in this context?

Reply:
Changed it to: "Reference to a parental object or legal person."

Reply:
"parent object". "parental" breaks the metaphor, I would only use that for an actual biological relationship.

Reply:
@apdavison thanks! I'll changed it.